docs/guides/logs.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: ""
+page_title: "Log Resource Examples"
+description: |-
+    Log Resource Examples
+---
+
+### Log Resource Examples
```

Comment:
Is it correct to start these guides with an h3? I'm unsure if there is specific formatting that requires this so flagging.

Reply:
Thanks for flagging that @brett0000FF, I'm just following the format of the only existing guide:
https://github.com/DataDog/terraform-provider-datadog/blob/master/docs/guides/monitors.md?plain=1

Whether it's correct might be a matter of opinion, but my goal for now is just to maintain consistency. An update to the heading structure might be a good follow up task, so I created DOCS-9226.